Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -138,6 +138,7 @@ static wxString xfb_virtual_desc = wxTRANSLATE("Emulate XFBs using GPU texture o
 static wxString xfb_real_desc = wxTRANSLATE("Emulate XFBs accurately.\nSlows down emulation a lot and prohibits high-resolution rendering but is necessary to emulate a number of games properly.\n\nIf unsure, check virtual XFB emulation instead.");
 static wxString dump_textures_desc = wxTRANSLATE("Dump decoded game textures to User/Dump/Textures/<game_id>/.\n\nIf unsure, leave this unchecked.");
 static wxString load_hires_textures_desc = wxTRANSLATE("Load custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave this unchecked.");
+static wxString cache_hires_textures_desc = wxTRANSLATE("Load custom textures on startup.\nThis may require much of RAM but fixes some stutter.\n\nIf unsure, leave this unchecked.");
```

Comment:
A little bikeshedding on the IRC turned up this - "Cache custom textures to system RAM on startup.  This can require exponentially more RAM but fixes possible stuttering. If unsure, leave this unchecked."